template/src/bin/main.rs

```diff
@@ -55,6 +55,8 @@ fn main() -> ! {
     esp_println::logger::init_logger_from_env();
     //ENDIF
 
+    esp_bootloader_esp_idf::esp_app_desc!();
```

Comment:
This doesn't need to go in `main`, it can go somewhere in the file as it expands just just a structure definition